everviz/plugins/crossplot/crossplot.py

```diff
@@ -35,10 +34,7 @@ def __init__(self, app, data_path, title="Crossplot"):
 
         ASSETS_DIR = pkg_resources.resource_filename("everviz", os.path.join("assets"))
 
-        WEBVIZ_ASSETS.add(os.path.join(ASSETS_DIR, "axis_customization.css"))
-        # We do this because webwiz currently doesn't give a way to serve
-        # css when running in non portable mode.
-        set_up_assets(app, ASSETS_DIR)
+        WEBVIZ_ASSETS.add(Path(os.path.join(ASSETS_DIR, "axis_customization.css")))
```

Comment:
If you use pathlib you can do: `Path(ASSETS_DIR) / "axis_customization.css"`, no need to mix Path and os.join